rosrider_fakenode:
  ros__parameters:
    wheels:
      separation: 0.1
      radius: 0.0685
    joint_states_frame: "base_footprint"
    odom_frame: "odom"
    base_frame: "base_footprint"
